recipes/minizip-ng/all/conanfile.py

```diff
@@ -90,7 +90,10 @@ def requirements(self):
             self.requires("openssl/[>=1.1 <4]")
         if self.settings.os != "Windows":
             if self.options.get_safe("with_iconv"):
-                self.requires("libiconv/1.17")
+                if Version(self.version) <= "4.0.7":
+                    self.requires("libiconv/1.17")
+                else:
+                    self.requires("libiconv/1.18")
```

Comment:
Please avoid this and leave it as it was, in order to avoid conflicts and having conditional logic in the recipe - we'll look into updating all currently maintained recipes to use libiconv 1.18 instead.

if you are using gcc15, you have a few options:
*  wait until https://github.com/conan-io/conan-center-index/pull/27839 is merged, this will fix the issue for libiconv/1.17
* add `tools.build:cflags=["-std=gnu17"]` in the `[conf]` section of your profile (you can also limit it to libiconv only if this is a concern)
* in the `[replace_requires]` section of your profile, add `libiconv/* : libiconv/1.18` - to force 1.18 wherever libiconv is referenced

Thanks!